### ISSUE TITLE
Support for Clients reading parquet

### ISSUE THREAD

**Issue by smaxwellstewart** [Parquet/parquet-format#106]
So I know this is nothing to do with this codebase directly. But does anyone know where to find clients that support reading (maybe even writing) parquet files in a variety of languages.

Seems right now the only people who support this scala as part of the spark world. Any other languages have libs / modules for handling these files?

**Comment by xhochy** [comment]
There is a C++ implementation https://github.com/apache/parquet-cpp and Python support through PyArrow: https://github.com/apache/arrow & https://pyarrow.readthedocs.io/en/latest/

**Comment by smaxwellstewart** [comment]
Yeah a golang implementation would be grand. It seems it's something people are working on but isn't stable yet.

**Comment by seufagner** [comment]
As well as @xhochy , do you can wrap C++ class in C and use it

https://stackoverflow.com/questions/1713214/how-to-use-c-in-go

**Comment by xhochy** [comment]
There is https://github.com/red-data-tools/parquet-glib which might be usable from Go.

**Comment by aloneguid** [comment]
We are almost done with .NET and .NET Core (C#) implementation, mostly in stabilisation phase now. https://github.com/elastacloud/parquet-dotnet. We support both reads and writes.
